src/plugins/folderView/components/GridItem.vue

```diff
@@ -69,7 +70,7 @@ export default {
     }
   },
   methods: {
-    navigate() {
+    navigate(event) {
```

Comment:
What is this event being used for?

Reply:
Looks like it's not. I'll change it to `_event` to notify it's available but unused

Reply:
Looks like it's not. I'll change it to `_event` to notify it's available but unused